trustfall_core/test_data/tests/valid_queries/fold_count_filter_with_overrode_filter.graphql.ron

```diff
@@ -0,0 +1,22 @@
+TestGraphQLQuery (
+    // Ensure that we properly handle two of the same type of filters
+    // but one of the filters beat's the other. In this case filtering
+    // by greater than two is beat by filtering for greater than three.
```

Comment:
```suggestion
    // but one of the filters dominates the other. In this case filtering
    // by greater than two is dominated by filtering for greater than three.
```

this is the technical term you were looking for here

also the apostrophe wasn't needed in the original